### ISSUE TITLE
iOS Multi-Destination Issue

### ISSUE THREAD

**Issue by bigdhri** [bitpay/copay#10813]
Hi Co Pay Team,
    I tried to do the Transfer to Multiple Recipients on the TestNet and the app errored out, with: Copay only supports Bitcoin Cash using new version numbers...
App Version: 9.3.4
The odd part is, if I do a single transaction to any of the addresses I want to send to, the application works correctly. 
Any help would be appreciated,
Bigdhri

**Comment by gabrielbazan7** [comment]
Yes, sorry about that, we will send a fix for that on the next release
